Effect for Complete Beginners/10. Introduction to Fibers.md

```diff
@@ -33,24 +29,23 @@ Note the `fiber=#0` and `fiber=#1` in the output. Effect tracks the fibers that
 To run an asynchronous task without blocking, we can use the `fork` operation:
 
 ```ts
-const program = Effect.gen(function* () {
-  yield* Effect.fork(
-    pipe(
-      Effect.sleep(3),
-      Effect.tap(() => Effect.log("Waited 3 seconds"))
-    )
+const program = Effect.gen(function*() {
+  yield* Effect.sleep("3 seconds").pipe(
+    Effect.andThen(Effect.log("Waited 3 seconds")),
+    Effect.fork
   );
 
   console.log("executed...");
-  yield* Effect.sleep(5);
```

Comment:
`Effect.sleep` takes a `DurationInput` as an argument. When the `DurationInput` is a number, it is treated as milliseconds, not seconds.